client/src/app/components/stock-detail-header/stock-detail-header.component.ts

```diff
@@ -15,16 +15,22 @@ export class StockDetailHeaderComponent {
   }
 
   stockChangeColor() {
-    if(this.stockInfo) {
-      return this.stockInfo.stockChange < 0 ? 'negative-change' : 'positive-change';
+    if (this.stockInfo) {
```

Comment:
This function can be cleaner by doing

`if (this.stockInfo && this.stockInfo.stockChange !==0)`

Since both these conditions lead to returning ' ' this is valid